serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/protobuf/ProtobufSerializer.java

```diff
@@ -25,6 +25,7 @@
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.Message;
 import com.squareup.wire.schema.internal.parser.ProtoFileElement;
+//import com.amazonaws.services.schemaregistry.utils.apicurio.FileDescriptorUtils;
```

Comment:
Same as above. Are we using the patched FileDescriptorUtils instead of the actual FileDescriptorUtils?